libdevcore/CommonData.h

```diff
@@ -120,6 +120,134 @@ inline void toBigEndian(T _val, Out& o_out)
 		o_out[i - 1] = (typename Out::value_type)(uint8_t)v;
 	}
 }
+#ifdef __INTEL_COMPILER
```

Comment:
What is the reason for needing these Intel-specific functions?   Where are they called from?